model/dns_zone/dns_server.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative "../../model"
+
+class DnsServer < Sequel::Model
+  many_to_many :dns_zones
+  many_to_many :vms
+
+  include ResourceMethods
+
+  def run_commands_on_all_vms(commands)
+    vms.each do |vm|
+      outputs = vm.sshable.cmd("sudo -u knot knotc", stdin: commands.join("\n")).split("\n")
```

Comment:
This is beyond the scope of this PR. However, remember that if one of the VMs is down, we won't be able to update any of them. Budding a new prog could be beneficial in this situation in the future.